tests/integration/test_design.py

```diff
@@ -489,6 +504,11 @@ def test_named_selections(modeler: Modeler):
     design.delete_named_selection("OnlyCircle")
     assert len(design.named_selections) == 3
 
+    # Try to vefify name selection using earlier backend version
+    design = modeler.open_file(Path(FILES_DIR, "25R1BasicBoxNameSelection.scdocx"))
+    hello = design.named_selections
+    assert hello[0].faces == []
+
```

Comment:
Could we have this in a separate test? I would like the whole test to be verified against the newest version of the backend and, by overriding the backend_version, the whole test now runs as if it was 24R2.